core/server/api/shared/pipeline.js

```diff
@@ -121,7 +121,7 @@ const pipeline = (apiController, apiUtils) => {
                 frame = new shared.Frame({
                     body: data,
                     options: options,
-                    context: {}
+                    context: options.context || {}
```

Comment:
should we omit context from `options` 🤔 ? 

> options: options,

Reply:
It doesn't hurt to still have it there, but on the other way its becoming a bit messy in the `options`... Wanted to zoom out and ask why do we need to create a `new Frame()` instance here instead of just doing:
```
                options.configure({
                    options: apiImpl.options,
                    data: apiImpl.data
                });
```
sinsce we already know that the instance if of a frame. This would relieave us from this copyin garound of unnecessary data :thinking:

Reply:
I would love to keep the originals clean and the frame expects `original.context`

Reply:
It feels kind of weird omitting any kind of properties here tbh. We don't really know what kind of data we have got in options and feels like it should be up to constructor to do data filtering (as that's who knows how it's using the data later)

Reply:
After discussing this further got the point :sweat_smile: Will omit context in options object